app/page/home/index.tsx

```diff
@@ -21,11 +33,11 @@ export function Home() {
   const hasNoAccount = useAccounts().length === 0
   return (
     <>
-      <Navbar />
       {hasNoAccount ? (
         <AccountCreation />
       ) : (
         <>
+          <Navbar />
           <AccountInfo />
           <Divider />
           <AccountNavigation />
```

Comment:
I think `AccountCreation` is more like a state logic, so it kind of weird that when `hasNoAccount` we render the AccountCreation logic (?) And it seems not the same level description compared to the latter.
Maybe we could just create account if we found out the user has no account in LN33.

Reply:
Yes, it is. Let me think about how to improve it! :runner: